app/src/main/java/bitmap/transformer/App.java

```diff
@@ -1,14 +1,73 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
+///*
+// * This Java source file was generated by the Gradle 'init' task.
+// */
 package bitmap.transformer;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.awt.Image;
+import java.io.File;
+import java.io.*;
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
 
 public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
-
     public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
+        if (args.length != 3) {
+            System.err.println("Usage: java BitmapTransformer <input_file> <output_file> <transform>");
+            return;
+        }
+        String inputFile = args[0];
+        String outputFile = args[1];
+        String transform = args[2];
+        try {
+            BufferedImage image = ImageIO.read(new File(inputFile));
+            BufferedImage transformedImage = null;
+            if (transform.equals("invert")) {
+                transformedImage = invertColors(image);
```

Comment:
Make sure the variables you're passing in are the same type the method is asking for